viewing_party/party.py

```diff
@@ -0,0 +1,129 @@
+#wave 1
+
+def create_movie(title, genre, rating):
+    if title ==  None:
+        return None
+    if genre == None:
+        return None
+    if rating == None:
+        return None
```

Comment:
since all of these if statements will produce the same return statement, let's combine them!

```suggestion
    if title ==  None or genre == None or rating == None:
        return None
```